ticdc/ticdc-open-api-v2.md

```diff
@@ -0,0 +1,1010 @@
+          "table_name": "string"
+        }
+      ],
+      "ignore_txn_start_ts": [
+        0
+      ],
+      "rules": [
+        "string"
+      ]
+    },
+    "force_replicate": true,
+    "ignore_ineligible_table": true,
+    "memory_quota": 0,
+    "mounter": {
+      "worker_num": 0
+    },
+    "sink": {
+      "column_selectors": [
+        {
+          "columns": [
+            "string"
+          ],
+          "matcher": [
+            "string"
+          ]
+        }
+      ],
+      "csv": {
+        "delimiter": "string",
+        "include_commit_ts": true,
+        "null": "string",
+        "quote": "string"
+      },
+      "date_separator": "string",
+      "dispatchers": [
+        {
+          "matcher": [
+            "string"
+          ],
+          "partition": "string",
+          "topic": "string"
+        }
+      ],
+      "enable_partition_separator": true,
+      "encoder_concurrency": 0,
+      "protocol": "string",
+      "schema_registry": "string",
+      "terminator": "string",
+      "transaction_atomicity": "string"
+    },
+    "sync_point_interval": "string",
+    "sync_point_retention": "string"
+  },
+  "create_time": "string",
+  "creator_version": "string",
+  "error": {
+    "addr": "string",
+    "code": "string",
+    "message": "string"
+  },
+  "id": "string",
+  "resolved_ts": 0,
+  "sink_uri": "string",
+  "start_ts": 0,
+  "state": "string",
+  "target_ts": 0,
+  "task_status": [
+    {
+      "capture_id": "string",
+      "table_ids": [
+        0
+      ]
+    }
+  ]
+}
+```
+
+The parameters are described as follows:
+
+| Parameter name | Description |
+|:-----------------|:---------------------------------------|
+| `admin_job_type`  | `INTEGER` type. The admin job type.                 |
+| `checkpoint_time` | `STRING` type. The formatted time of the current checkpoint for the replication task.                  |
+| `checkpoint_ts`   | `STRING` type. TSO of the current checkpoint for the replication task.    |
+| `config`          | The replication task configuration. The structure and meaning are the same as that of the `replica_config` configuration in creating the replication task.       |
+| `create_time`     | `STRING` type. The time when the replication task is created.                          |
+| `creator_version` | `STRING` type. The TiCDC version when the replication task is created.         |
+| `error`           | The replication task error.                      |
+| `id`              | `STRING` type. The replication task ID.                |
+| `resolved_ts`     | `UINT64` type. The replication task resolved ts.    |
+| `sink_uri`        | `STRING` type. The replication task sink URI.                                     |
+| `start_ts`        | `UINT64` type. The replication task start ts.                                      |
+| `state`           | `STRING` type. The replication task status. It can be `normal`, `stopped`, `error`, `failed`, and `finished`. |
```

Comment:
```suggestion
| `state`           | `STRING` type. The replication task status. It can be `normal`, `stopped`, `error`, `failed`, or `finished`. |
```